src/frontend/pages/LandingPage/LandingPage.js

```diff
@@ -1,9 +1,18 @@
 import React from "react";
+import { Link } from "react-router-dom";
+
+import "./LandingPage.css"
 
 const LandingPage = () => {
     return (
         <>
```

Comment:
A similar suggestion here as well. as <div> is already a parent here, I don't see the usecase of a fragment,

Reply:
would keep this in mind further